### PR TITLE
Help/About: Update the About page for 6.6.

### PR DESCRIPTION
**Pull request by ryelle**
This PR introduces the new content for the 6.6 About page. This release, the About page will have just the highlights for 6.6, and link off to the release page on WordPress.org for the full overview. See https://github.com/WordPress/wporg-main-2022/issues/454 for the release page tracking. Also see https://github.com/WordPress/gutenberg/issues/62631 for more discussion on this.

| Desktop | Small screen |
|---|---|
| large-screen | small-screen |

Trac ticket: https://core.trac.wordpress.org/ticket/61320

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by ryelle** [comment]
Based on feedback https://github.com/WordPress/gutenberg/issues/62631#issuecomment-2187678287, I've combined the subheadings & paragraphs into one paragraph:

Screen Shot 2024-06-24 at 20 44 43

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -53,172 +53,115 @@
...
-				<p><?php _e( 'Set aspect ratios for Cover block images and easily add color overlays that automatically source color from your chosen image.' ); ?></p>
 			</div>
-			<div class="column">
-				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.5/6-box-shadow.webp" alt="" height="270" width="270" />
-				</div>
-				<h3 class="is-smaller-heading" style="margin-bottom:calc(var(--gap) / 4);"><?php _e( 'Add box shadow supports to even more blocks' ); ?></h3>
-				<p><?php _e( 'With shadow supports enabled, you can create layouts with visual depth or add a little personality to your design.' ); ?></p>
+			<div class="column is-vertically-aligned-center">
+				<h3><?php _e( 'Get more details from your style revisions' ); ?></h3>
```
Comment (andrewserong) [inline src/wp-admin/about.php]: Just a drive-by comment, so apologies if this feedback isn't right, but this appears to be the old heading from WP 6.5? The body of this feature refers to the new layout for navigating and editing pages in the site editor, but the heading here refers to style revisions.

Comment (andrewserong) [reply src/wp-admin/about.php]: From a look at the doc linked in https://github.com/WordPress/gutenberg/issues/62631#issuecomment-2187313214, it looks like the heading is meant to be:

```suggestion
				<h3><?php _e( 'Quick previews for pages' ); ?></h3>
```

Comment (jasmussen) [reply src/wp-admin/about.php]: Good catch, Andrew is right!

Comment (ryelle) [reply src/wp-admin/about.php]: Aha, yes! this is why I was asking for a second look 🙂  Thanks!

**Comment by ryelle** [comment]
Committed in [r58568](https://core.trac.wordpress.org/changeset/58568).
